src/Drupal/Driver/DrushDriver.php

```diff
@@ -189,11 +189,27 @@ public function clearStaticCaches() {
   }
 
   /**
+   * Decodes JSON output returned by Drush.
+   *
+   * It will clean up any junk that may have appeared before the JSON.
+   *
+   * @param string $output
+   *   The output from Drush.
+   * @return object
+   *   The decoded JSON object.
+   */
+  protected function decodeJsonOutput($output) {
+    // Remove anything before the first '{'.
+    $output = preg_replace('/^[^\{]*/', '', $output);
```

Comment:
Ack, yeah, probably... That's a harder regex to write, though - I'm not sure how to make sure it goes from the last `}` and not the first `}`. I suppose we could reverse the order of the string, check from the beginning, and then reverse it back to the original order? Anyone know regexes better than me know a way to do it?